app/controllers/comments_controller.rb

```diff
@@ -5,15 +5,30 @@ def new
     @post = @user.posts.find(params[:post_id])
   end
 
+  def index
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @comments = @post.comments
+    respond_to do |format|
+      format.json { render json: @comments }
+    end
+  end
```

Comment:
- [ ] Your comment API ` - GET : /users/:user_id/posts/:id/comments` and `POST :/users/:user_id/posts/:id/comments` is working but I do not see the `json` response code ` format.json { render json: @posts }` in your post_controller. Kindly fix this

Reply:
i have added it to the controller thanks for the suggestion

Reply:
Please Request for review from your dashboard.